.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: 🐜 Bug Report
+about: Create a report to help us improve
+title: '[bug] '
+labels: bug
+---
+
+<!-- Please make sure you are posting an technical issue related to Gridcoin. --> 
+
+<!-- For general questions about Gridcoin or wallet recovery please use one of the various communities:
+* [Gridcoin on reddit](https://www.reddit.com/r/gridcoin/)
+* [Discord](https://discord.gg/jf9XX4a) -->
+
+<!-- ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION. -->
+
+# Bug Report
+
+**Current behavior:**
+<!-- Describe how the bug manifests. -->
+
+**Expected behavior**
+<!-- Describe what the behavior would be without the bug. -->
+
+**Steps to reproduce:**
+<!--  Please explain the steps required to duplicate the issue, especially if you are able to provide a sample application or sample code -->
```

Comment:
Let's just leave it at "Please explain the steps required to duplicate the issue." Developers that are code aware will know to include code snippets. I don't necessarily want imply that we require non-programmers to actually do work in the code itself.